.github/workflows/audit.yml

```diff
@@ -0,0 +1,33 @@
+name: Audit
+
+on:
+  push:
+  schedule:
+    - cron: "40 10 * * *"
+
+jobs:
+  audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - uses: actions/setup-node@v2.5.1
+        with:
+          node-version: "16.x"
+          registry-url: "https://registry.npmjs.org"
+      - run: npm ci
+      - run: npm run licenses:check
+      # Some issues identified by npm audit cannot be fixed by us
+      # (i.e. they require an upstream dependency update),
+      # but using `npm-force-resolutions` we can at least override the upstream
+      # dependency when running an audit.
+      # If this task fails, first attempt to run `npm audit fix` or upgrade the
+      # relevant dependency; when that does not work, you can add the transitive
+      # dependency to the `resolutions` field in package.json according to the
+      # instructions in `npm-force-resolutions`'s README.
+      - run: npx npm-force-resolutions && npm audit --audit-level=moderate
```

Comment:
@inrupt/sdk-developers we could also run this with only `--production` which is the dependencies consumers of our code will get.

Reply:
This could be useful if the audit becomes an issue, but if we don't have an audit for dev dependencies, couldn't this compromise the built code (like, in a very worst case scenario) ?

Reply:
Yeah, I was more thinking for production dependencies, we'd wanna not use npm-force-resolutions, but for dev dependencies use it.